.coafile

```diff
@@ -117,7 +117,7 @@ bears = reSTLintBear
 files = README.rst
 
 [text]
-bears = SpaceConsistencyBear
+bears = SpaceConsistencyBear, YAMLLintBear
```

Comment:
@mykromo Please create a different section for `yml` files, which will only check for yml files.
Your current changes are using `YAMLLintBear` even on `.sh`, `.cfg`, `.css` files which it shouldn't

Reply:
I see. Alright. Thanks!